README.md

```diff
@@ -1,35 +1,116 @@
-## How it works
+# Connect
 
-The Connectivity component is responsible for the communication with the target system. This component implements the logic of the interaction protocol, receives and sends messages from and to the system, respectively.
```

Comment:
This component **is responsible for implementing the logic of the interaction protocol and for sending and receiving messages from/to the system respectively.**